Sources/KlaviyoSwift/Vendor/ComposableArchitecture/Cancellation.swift

```diff
@@ -0,0 +1,309 @@
+//
```

Comment:
This is vendor code from https://github.com/pointfreeco/swift-composable-architecture. I mostly left it as is except for excluding dependencies. I'm going to add a todo with a readme on why this is here and not fully included. Potentially in the future we can have this as a separate target or even package.